part_2.md

```diff
@@ -89,12 +89,9 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
-You will also need to import the relevant libraries, by adding to the top of your `Startup.cs` file:
-```csharp
-using Microsoft.AspNetCore.Authentication.JwtBearer;
-using Microsoft.Identity.Web;
-using System.IdentityModel.Tokens.Jwt;
-```
+You will also need to import the relevant libraries. VSCode will fix this for you automatically if you click on the missing import (highlighted in red) and then press Ctrl/Cmd + FullStop:

```

Comment:
I believe this is only for auto-adding imports when using completions rather than a fixing a problem discovered by the language server. This is still really useful though and we should definitely include it:
```suggestion
You will also need to import the relevant libraries. VSCode will fix this for you automatically if you click on the missing import (highlighted in red) and then press Ctrl/Cmd + FullStop. Note that if you turn on the 'Enable Import Completion' setting on (File -> Preferences -> Settings -> Extensions -> C# -> Omnisharp: Enable Import Completion) then these imports will be added for automatically when selecting a completion option.
```